contrib/opencensus-ext-azure/README.rst

```diff
@@ -73,6 +73,18 @@ You can enrich the logs with trace IDs and span IDs by using the `logging integr
         logger.warning('In the span')
     logger.warning('After the span')
 
+You can also add custom properties to your log messages in the form of key-values.
+
```

Comment:
You should add a warning noting that string formatting will be lost when using custom properties. There doesn't seem to be any way to avoid this.

Reply:
I tested this and if the logging instruction includes a dictionary and string formatting, like this `logger.warning("test_%s", { "SKU" : "SKU-1" })` the message will be logged like this: `test_{'SKU': 'SKU-1'}` and the dictionary would be logged as well inside the `customProperties` column.

Reply:
That's true but the common usage of the args parameter is using a string variable. If user's logging messages are already using string formatting, you are forcing them to change to passing in dictionaries.

Reply:
Added the warning message.